conf/gconf/cluster.conf.sample

```diff
@@ -0,0 +1,296 @@
+# This is CORTX GConf (Config Store) which serves as input 
+# to all the CORTX Services
+# Location: /etc/cortx/cluster.conf 
+
+cortx:
+  external:
+    consul:
+      admin: admin
+      endpoints:
+      - http://consul-server.cortx-cluster.com:80
+      secret: MG12NU5BVjBVeWpBSWJuSV9uVjV5UzUxN3lTSVNkNUk1YkRzbjd2QkE9PQ==
+    kafka:
+      admin: admin
+      endpoints:
+      - kafka-server1.cortx-cluster.com
+      - kafka-server2.cortx-cluster.com
+      secret: TmRPNThMSm5GOVl5RHgwTVFfaTRJZ3k3YUUwMldmNkdxRE1KYVIxc0E9PQ==
+      
+  common:
+    product_release: LC              # TBD: To be removed
+    security:
+      device_certificate: /etc/cortx/solution/ssl/stx.pem
+      domain_certificate: /etc/cortx/solution/ssl/stx.pem
+      ssl_certificate: /opt/seagate/cortx/s3/install/haproxy/ssl/s3.seagate.com.pem
+    service:
+      admin: admin
+      secret: Z0FBQUFBQmhVY1dCdzJ6RkRXT3FSVjJXTWtUM0paZlB1LXFRRW0xNy1rRGhJLVpHUjcxMkVOeDZ6
+        MG9KWmpsM21yTmprZTJkYTBHUU1OWVl4dnZSaVRTeEs4UEdReHhkR1E9PQ==
+    setup_size: small                # TBD: To be removed and derived from limits
+    setup_type: K8                   # To be removed
+    storage:
+      config: /etc/cortx
+      local: /etc/cortx
+      log: /var/log/cortx/
+      
+  csm:
+    agent:
+      endpoints:
+      - https://csm.seagate.com:8081
+    email_address: cortx@seagate.com # Optional 
+    mgmt_admin: cortxadmin
+    mgmt_secret: UlU1U0NrYklraS0ta3BOamN5X2J1VmUwRnJXTUFlb2JDdkItY1B3ZEE9PQ==
+    limits:
+      num_services: 1
+      services:
+      - name: agent
+        memory:
+          min: 128Mi
+          max: 256Mi
+        cpu:
+          min: 250m
+          max: 500m
+  ha:
+    limits:         
+      services:        
+      - name: ha
+        memory:
+          min: 128Mi
+          max: 1Gi
+        cpu:
+          min: 250m
+          max: 500m  
+  rgw:
+    auth_user: auth_user            # auth admin user name
+    auth_admin: auth_access_key     # auth admin user access key
+    auth_secret: TVRGNDBQVnY3OXJ5bG4tLTQwSjNZQ0xQZVJiczBmUVNqcXFvTXdvNHc9PQ==
+    data:
+      endpoints:
```

Comment:
data keyword adds confusion to be exposed as data for external clients. Ideally we require following ports to be exposed
data:        # list of all endpoints which will be available for data transfer and can be consumed by external loadbalancer. We should discuss if this is at all required.
     endpoints: 
            - https://pod1:443
            - https://pod2:443
internal: # will be used for internal communication within cortx like csm-s3 communication
            - https://cortx-io-svc:28000

Reply:
@ujjwalpl please confirm on the above keys

Reply:
discussed with ujjwal and as of now we are not making this change.

Reply:
@kailashyadav11 Lets have the following 
```
  rgw:
    ...
    s3:
      endpoints:
      - http://s3.seagate.com:80
```

Reply:
changed the "data" endpoints to "s3" endpoints
s3:
      endpoints:
      - http://s3.seagate.com:80
      - https://s3.seagate.com:443

Reply:
> [http://s3.seagate.com:80](http://s3.seagate.com/)
[https://s3.seagate.com:443](https://s3.seagate.com/)

these are good, but too generic i feel, product name in domain name would avoid colliding with other s3 endpoints from seagate, if at all.

Can be 
[http://s3.cortx.seagate.com:80](http://s3.cortx.seagate.com/)
[https://s3.cortx.seagate.com:443](https://s3.cortx.seagate.com/)